Meadowlark-js/tests/e2e/scenarios/ResourcesGetValidation.test.ts

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { getAccessToken } from '../helpers/Credentials';
+import { createContentClassDescriptor } from '../helpers/DataCreation';
+import { deleteResourceByLocation } from '../helpers/Resources';
+import { baseURLRequest, rootURLRequest } from '../helpers/Shared';
+
+describe('given a GET of an existent EducationContent', () => {
+  let contentClassDescriptorLocation = '';
+  let educationContentLocation = '';
+  const educationContentEndpoint = 'EducationContents';
+
+  const educationContentBody = {
+    contentIdentifier: '933zsd4350',
+    namespace: '43210',
+    shortDescription: 'abc',
+    contentClassDescriptor: 'uri://ed-fi.org/ContentClassDescriptor#Presentation',
+    learningResourceMetadataURI: '21430',
+  };
+
+  beforeAll(async () => {
+    contentClassDescriptorLocation = await createContentClassDescriptor();
+
+    await baseURLRequest()
+      .post(`/v3.3b/ed-fi/${educationContentEndpoint}`)
+      .send(educationContentBody)
+      .auth(await getAccessToken('host'), { type: 'bearer' })
+      .expect(201)
+      .then((response) => {
+        expect(response).not.toBe('');
+        educationContentLocation = response.headers.location;
+      });
+  });
+
+  it('should return the EducationContent', async () => {
+    await rootURLRequest()
+      .get(educationContentLocation)
+      .auth(await getAccessToken('host'), { type: 'bearer' })
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual(expect.objectContaining(educationContentBody));
+      });
+  });
+
+  afterAll(async () => {
+    await deleteResourceByLocation(educationContentLocation);
+    await deleteResourceByLocation(contentClassDescriptorLocation);
```

Comment:
Note how the `ResourceGetValidation` test is also testing _delete_ functionality! and Of course it tested the `post` functionality.

We could probably have a single CRUD scenario like this:
```
describe when performing crud operations
  let location;

  describe when creating a new resource
    let getResponse;

    beforeAll
        location = ...
        // perform the post here and set `getResponse` in the `then` handler

    it returns 201
        expect(getResponse.status).toBe(201);


  describe when getting a resource by ID
    describe given the resource exists
      // Do a GET request with the location

    describe given the resource doe not exist
      // Do a GET request with a modified version of the location - for example, change the last character in the location string

  describe when getting all resources
    // expect 1 resource

  describe when upserting a resource
      // Run the post again and make sure it returns 200

  describe when updating a resource
      // Run the put with at least one property modified, and make sure it returns 204
      // Then run another GET by ID and make sure the value changed appropriately

  describe when  deleting a resource
      // Run the delete and make sure it returns 204
      // Then run another GET by ID and make sure it return 404
```

Reply:
Just to confirm. I can create just one scenarios to cover all the CRUD methods. And I can use the Descriptor, no need to use the `EducationContent` resource. 
Is this correct?